.rubocop.yml

```diff
@@ -51,6 +51,9 @@ Lint/AmbiguousBlockAssociation:
   Exclude:
     - spec/models/review_policy_class_spec.rb
 
+Naming/VariableNumber:
+  Enabled: false
```

Comment:
Is it necessary to disable this completely? If we don't want to use `rubocop:disable` comments, perhaps only disable it for `spec/**/*`?